packages/payment-backend/src/service.ts

```diff
@@ -264,3 +326,34 @@ export const createPaymentFromUnpaidOrder = async (p: {
   }
   return createPaymentFromOrder(p)
 }
+
+export const createAuthorizedPaymentAndGetCardUpdateUrl = async (
+  order: Order
+): Promise<String> => {
+  checkBackendUrlExists()
+
+  const url = `${process.env.PAYMENT_BACKEND_URL}/payment-admin/online/create/card-renewal-payment`
+
+  const dto = {
+    order: {
+      order: {
+        ...order,
+        customerFirstName: order.customer?.firstName,
+        customerLastName: order.customer?.lastName,
+        customerEmail: order.customer?.email,
+      },
+      items: order.items,
+    },
+  }
+
+  try {
+    const result = await axios.post<String>(url, dto)
+    return result.data
+  } catch (e) {
+    throw new ExperienceFailure({
+      code: 'failed-to-cancel-subscription',
```

Comment:
Why is this failed to cancel subscription error when the method is createAuthorizedPaymentAndGetCardUpdateUrl?

Reply:
Oh I just copied and didnt change this, good catch!